x-pack/plugins/security_solution/public/management/cypress/support/e2e.ts

```diff
@@ -102,7 +118,61 @@ Cypress.Commands.add(
 
 Cypress.on('uncaught:exception', () => false);
 
-// Login as a Platform Engineer to properly initialize Security Solution App
+Cypress.Commands.add('createEndpointHost', () =>
+  cy.dataSession(
+    ENDPOINT_HOST_SESSION_NAME,
+    () => {
+      let indexedPolicy: IndexedFleetEndpointPolicyResponse;
+      let policy: PolicyData;
+      let createdHost: CreateAndEnrollEndpointHostResponse;
+
+      return getEndpointIntegrationVersion().then((version) =>
+        createAgentPolicyTask(version).then((data) => {
+          indexedPolicy = data;
+          policy = indexedPolicy.integrationPolicies[0];
+
+          return enableAllPolicyProtections(policy.id).then(() => {
+            // Create and enroll a new Endpoint host
+            return createEndpointHost(policy.policy_id).then((host) => {
+              createdHost = host as CreateAndEnrollEndpointHostResponse;
+
+              return {
+                indexedPolicy,
+                policy,
+                createdHost,
+              };
+            });
+          });
+        })
+      );
+    },
+    true // don't invalidate the session when the test rerenders
```

Comment:
we can add a more advanced invalidation method, but in its current shape it should already improve the dev experience when running tests locally